packages/bierzo-wallet/src/routes/payment/components/CurrencyToSend/index.tsx

```diff
@@ -20,13 +24,86 @@ const useStyles = makeStyles((theme: Theme) => ({
   },
 }));
 
-export const CurrencyToSend = () => {
+const QUANTITY_FIELD = 'quantityField';
+const QUANTITY_MAX = 9999999;
+const QUANTITY_MIN = 1e-9;
+const CURRENCY_FIELD = 'currencyField';
+
+interface Currencies {
+  [key: string]: number;
+}
+
+//NOTE hardcoded placeholder balances for each currency
+const currencies: Currencies = {
+  ALT: 15,
+  IOV: 24,
+};
+
+const currencyItems = Object.keys(currencies).map(currency => {
+  const item: Item = {
+    name: currency,
+  };
+
+  return item;
+});
+
+const onSubmit = () => {};
+
+const CurrencyToSend = () => {
   const classes = useStyles();
 
   const avatarClasses = {
     root: classes.avatar,
   };
 
+  //NOTE hardcoded initial state for "currency"
+  const [currency, setCurrency] = useState(currencyItems[1].name);
```

Comment:
Please put all hooks once after other, meaning move down the avatarClasses. In this way is easier for the eye and harder to introduce bugs.

Reply:
Fixed by [99da2af](https://github.com/iov-one/ponferrada/pull/124/commits/99da2af71c58070eed2dffe202fc5b44feb007da).